corehq/apps/app_manager/fixtures/mobile_ucr.py

```diff
@@ -60,7 +60,7 @@ def uses_reports(self, restore_state):
             return False
 
         apps = self._get_apps(restore_state, restore_user)
-        report_configs = self._get_report_configs(apps).values()
+        report_configs = list(self._get_report_configs(apps).values())
         if not report_configs:
             return False
```

Comment:
Probably not something you want to fix now, but this could be simplified to
```py
return bool(self._get_report_configs(apps))
```
 The `.values()` call is not necessary for how the result is being used.

Reply:
https://github.com/dimagi/commcare-hq/pull/19033